### ISSUE TITLE
When a message is scheduled, personalization cannot be retrieved by using Jobs or Notifications API

### ISSUE THREAD

**Issue by heyitsmebev** [GSA/notifications-api#912]
When a user schedules a message, the confirmation status page is unable to display a preview of the message due to the absence of associated personalization. It appears that a notification_id is not generated and stored in the Jobs table until the message is sent. Personalization seems linked solely to the Notifications table and not to Jobs. As a result, when a message is scheduled, there is no personalization available for retrieval.

![Image](https://github.com/GSA/notifications-admin/assets/53159604/1ad7be2e-c825-463b-a72b-521bbf24ebb7)

**Comment by A-Shumway42** [comment]
@terrazoon Yeah, I believe this connected. Both this issue and the calculation of the message parts with personalization to show users could be taken care of if we separate those processes.

**Comment by ccostino** [comment]
Is it possible that this is also related to this error being thrown in the API? https://github.com/GSA/notifications-api/issues/1136

**Comment by ccostino** [comment]
Noting here that without these two things split, that's why we're sometimes seeing stack traces like what was noted in https://github.com/GSA/notifications-api/issues/1134.

**Comment by ecayer** [comment]
Moving back to up-next, because we believe this is unblocked?

**Comment by terrazoon** [comment]
I actually can't reproduce this:

1. schedule a one-off message to go out tomorrow at noon
2. preview shows personalization as expected.

**Comment by heyitsmebev** [comment]
This ticket appears to be obsolete, as I’m unable to reproduce the issue and everything works as expected.

**Comment by ccostino** [comment]
Thank you both for double checking and confirming!  We'll close this one out.
